examples/example_x11_vulkan/main.cpp

```diff
@@ -0,0 +1,592 @@
+                    if(((xcb_client_message_event_t*)event)->data.data32[0] == wm_delete_window)
+                        done = true;
+                    break;
+                }
+                case XCB_CONFIGURE_NOTIFY: {
+                    xcb_configure_notify_event_t *config = (xcb_configure_notify_event_t*)event;
+                    // Set DisplaySize here instead of checking in X11 NewFrame
+                    // Checking window size is request/response
+                    g_SwapChainResizeWidth = config->width;
+                    g_SwapChainResizeHeight = config->height;
+                    g_MainWindowData.FrameIndex = 0;
+                    ImGui::GetIO().DisplaySize = ImVec2(config->width, config->height);
+                    break;
+                }
+                default:
+                    break;
+                }
+            }
+            // xcb allocates the memory for the event and specifies
+            // the user to free it
+            free(event);
+        }
+        else
+        {
+            // Resize swap chain?
+            if (g_SwapChainRebuild && g_SwapChainResizeWidth > 0 && g_SwapChainResizeHeight > 0)
+            {
+                g_SwapChainRebuild = false;
+                ImGui_ImplVulkan_SetMinImageCount(g_MinImageCount);
+                ImGui_ImplVulkanH_CreateOrResizeWindow(g_Instance, g_PhysicalDevice, g_Device, &g_MainWindowData, g_QueueFamily, g_Allocator, g_SwapChainResizeWidth, g_SwapChainResizeHeight, g_MinImageCount);
+                g_MainWindowData.FrameIndex = 0;
+            }
+            // Start the Dear ImGui frame
+            ImGui_ImplVulkan_NewFrame();
+            ImGui_ImplX11_NewFrame();
+            ImGui::NewFrame();
+            // 1. Show the big demo window (Most of the sample code is in ImGui::ShowDemoWindow()! You can browse its code to learn more about Dear ImGui!).
+            if (show_demo_window)
+                ImGui::ShowDemoWindow(&show_demo_window);
+
+            // 2. Show a simple window that we create ourselves. We use a Begin/End pair to created a named window.
+            {
+                static float f = 0.0f;
+                static int counter = 0;
+
+                ImGui::Begin("Hello, world!");                          // Create a window called "Hello, world!" and append into it.
+
+                ImGui::Text("This is some useful text.");               // Display some text (you can use a format strings too)
+                ImGui::Checkbox("Demo Window", &show_demo_window);      // Edit bools storing our window open/close state
+                ImGui::Checkbox("Another Window", &show_another_window);
+
+                ImGui::SliderFloat("float", &f, 0.0f, 1.0f);            // Edit 1 float using a slider from 0.0f to 1.0f
+                if(ImGui::ColorEdit3("clear color", (float*)&clear_color)) // Edit 3 floats representing a color
+                {
+                    wd->ClearValue.color.float32[0] = clear_color.x;
```

Comment:
I think all the Vulkan examples are currently incorrect with the background color. The ClearValue is never set therefore it's always black. If we set the ClearValue when this widget changes then it works properly. I can create a separate PR fixing these.